_documentation/user/flags/ricochet.md

```diff
@@ -5,4 +5,12 @@ type: Good
 summary: Shots bounce off walls. Don't shoot yourself!
 ---
 
-...
+The Ricochet (R) flag allows the player to fire shots that can bounce off of the objects they hit. The direction of the bounce is determined by the angle under which the platform is hit. An exception are objects with the attributes `shootthrough` or `passable`, where the result is the bullet passing through the object.
+
+A lot of maps utilize the default ricochet option to already allow the effect of Ricochet without the tanks being required to hold the specific flag for it. The Ricochet flag holds meaning and effect on maps that do not utilize said option.
+
+## Tactics
+
+Ricochet is potent on maps filled with objects, usually boxes, behind which a tank can hide from direct enemy fire, all while attempting to shoot them down by using the objects as proxies off of which own bullets bounce in hopes of hitting the opponent.
+
+Efficiently utilizing the Ricochet flag requires a high amount of estimating the bullet's trajectory by planning the path before it is shot, usually through the use of radar. Complexity behind the estimation increases with each additional bounce required to reach the target.
```

Comment:
"Efficiently" is borderline subjective. Consider a more declarative style
```suggestion
Effective use of the Ricochet flag requires accurate estimation of the path of a shot after it bounces off an obstacle. Multiple bounces compounds the difficulty. One may observe the trajectory of a shot and adjust the point of aim to increase accuracy.
```

Reply:
@jwmelto agreed, will change.